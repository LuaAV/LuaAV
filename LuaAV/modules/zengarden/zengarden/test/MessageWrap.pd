#N canvas 588 22 450 300 10;
#X obj 45 27 loadbang;
#X obj 114 28 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 377 219 wrap;
#X obj 342 283 print;
#X obj 313 219 wrap 4.24;
#X msg 336 168 2.54;
#X msg 173 168 -13;
#X msg 204 168 1;
#X msg 85 168 6.2 3;
#X obj 45 68 t b b b b b b b b b b b b b b b b b b b b b b b;
#X msg 16 169 0;
#X msg 377 167 13.31;
#X msg 261 168 11.44;
#X msg 47 169 -5.7;
#X msg 129 168 3 6.2;
#X msg 304 168 8;
#X obj 202 223 wrap 3 10;
#X msg 233 168 -5;
#X connect 0 0 9 0;
#X connect 1 0 9 0;
#X connect 2 0 3 0;
#X connect 4 0 3 0;
#X connect 5 0 4 0;
#X connect 6 0 16 0;
#X connect 7 0 16 0;
#X connect 8 0 16 1;
#X connect 9 0 10 0;
#X connect 9 1 6 0;
#X connect 9 2 7 0;
#X connect 9 3 17 0;
#X connect 9 4 12 0;
#X connect 9 5 13 0;
#X connect 9 6 6 0;
#X connect 9 7 7 0;
#X connect 9 8 17 0;
#X connect 9 9 12 0;
#X connect 9 10 8 0;
#X connect 9 11 6 0;
#X connect 9 12 7 0;
#X connect 9 13 17 0;
#X connect 9 14 12 0;
#X connect 9 15 14 0;
#X connect 9 16 6 0;
#X connect 9 17 7 0;
#X connect 9 18 17 0;
#X connect 9 19 12 0;
#X connect 9 20 15 0;
#X connect 9 21 5 0;
#X connect 9 22 11 0;
#X connect 10 0 16 0;
#X connect 11 0 2 0;
#X connect 12 0 16 0;
#X connect 13 0 16 1;
#X connect 14 0 16 1;
#X connect 15 0 4 0;
#X connect 16 0 3 0;
#X connect 17 0 16 0;
