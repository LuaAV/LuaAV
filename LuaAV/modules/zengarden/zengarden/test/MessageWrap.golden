[@ 0.000ms] print: 0.31
[@ 0.000ms] print: 2.54
[@ 0.000ms] print: 3.76
[@ 0.000ms] print: 4.44
[@ 0.000ms] print: 9
[@ 0.000ms] print: 8
[@ 0.000ms] print: 8
[@ 0.000ms] print: 5.04
[@ 0.000ms] print: 4.6
[@ 0.000ms] print: 4.2
[@ 0.000ms] print: 6.2
[@ 0.000ms] print: 5.04
[@ 0.000ms] print: 4.6
[@ 0.000ms] print: 4.2
[@ 0.000ms] print: 6.2
[@ 0.000ms] print: -5.66
[@ 0.000ms] print: -5
[@ 0.000ms] print: -4.7
[@ 0.000ms] print: -1.6
[@ 0.000ms] print: -5.7